sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -319,6 +332,70 @@ export class StudentService extends RecordDataModelService<Student> {
     return null;
   }
 
+  /**
+   * Check for a partial match for a student in SFAS.
+   * @param user user to be matched against.
+   * @param student student to be matched against.
+   * @param studentSIN sin to be matched against.
+   * @param auditUserId audit user for the notification.
+   * @param externalEntityManager entityManager to be used to perform the query.
+   */
+  private async getStudentPartialMatch(
+    user: User,
+    student: Student,
+    studentSIN: string,
+    auditUserId: number,
+    externalEntityManager: EntityManager,
+  ) {
+    let partialMatch: SFASIndividual;
+    try {
+      partialMatch =
+        await this.sfasIndividualService.getIndividualStudentPartialMatch(
+          user.lastName,
+          student.birthDate,
+          studentSIN,
+        );
+
+      if (partialMatch) {
+        // Send out a notification of a partial match for the student information.
+        let matches = "";
+        if (
+          user.lastName.toLocaleLowerCase() ===
+            partialMatch.lastName.toLocaleLowerCase() &&
```

Comment:
There was no need so far to use `toLocaleLowerCase` instead of the `toLowerCase`. If there is a benefit for our scope, could you please clarify it?

Reply:
updated